src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/AuditIntegrationTest.kt

```diff
@@ -48,7 +48,6 @@ class AuditIntegrationTest : IntegrationTestBase() {
 
   @Test
   @Sql(
-    "classpath:test_data/clear-all-licences.sql",
     "classpath:test_data/clear-audit-events.sql",
```

Comment:
Not sure why the clear-all-licences has been removed? It guarantees that data is cleared down for any test order.